test.py

```diff
@@ -0,0 +1,2 @@
+l = {1,2}
+l.update(range(4))
```

Comment:
Accidental commit?

Reply:
It's merely a test file, I'll delete it.

Reply:
fixed : b2ac0cb